dotCMS/src/main/java/com/dotcms/rest/api/v1/index/ESIndexResource.java

```diff
@@ -483,16 +483,22 @@ public Response getActive(@Context HttpServletRequest httpServletRequest, @Conte
     @GET
     @Path("/docscount/{params:.*}")
     @Produces("text/plain")
-    public Response getDocumentCount(@Context HttpServletRequest httpServletRequest, @Context final HttpServletResponse httpServletResponse, @PathParam("params") String params) {
+    public Response getDocumentCount(@Context HttpServletRequest httpServletRequest,
+            @Context final HttpServletResponse httpServletResponse,
+            @PathParam("params") String params)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'params' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5171086)